channel-jira-server/src/main/java/com/synopsys/integration/alert/channel/jira/server/JiraServerProperties.java

```diff
@@ -40,6 +42,7 @@ public JiraServerRestConfig createJiraServerConfig() throws IssueTrackerExceptio
         jiraServerConfigBuilder.setAuthPassword(password);
         jiraServerConfigBuilder.setAuthUsername(username);
         jiraServerConfigBuilder.setProxyInfo(proxyInfo);
+        jiraServerConfigBuilder.setTrustCert(trustCert);
```

Comment:
What's this about?

Reply:
This was added as jira server wouldn't use the trustcert value passed into Alert when communicating causing pkix errors. Seemed like a bug to me as well as one that was preventing me from properly testing this functionality.